testcases/kernel/syscalls/sighold/sighold02.c

```diff
@@ -102,6 +103,7 @@ int main(int ac, char **av)
 	setup();
 
 	for (lc = 0; TEST_LOOPING(lc); lc++) {
+#if CODECOMMENT
```

Comment:
Please just use `#if 0`.  Everyone (including most editors) knows that this means disabled code.  Having to go and look for the definition of `CODECOMMENT` makes it harder to read.

Reply:
Code modifications are done.